src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -19,21 +19,59 @@
  * MA 02110-1301, USA.
  */
 
-#include <primitives/GTMenu.h>
-#include <primitives/PopupChooser.h>
-
 #include "GTTestsRegressionScenarios_7001_8000.h"
+
+#include <QFileInfo>
+
 #include "GTUtilsMdi.h"
 #include "GTUtilsMsaEditor.h"
 #include "GTUtilsMsaEditorSequenceArea.h"
 #include "GTUtilsTaskTreeView.h"
+
+#include "primitives/GTMenu.h"
+#include "primitives/GTWidget.h"
+#include "primitives/PopupChooser.h"
+
+#include "runnables/ugene/corelibs/U2Gui/AppSettingsDialogFiller.h"
 #include "runnables/ugene/corelibs/U2View/ov_msa/ExtractSelectedAsMSADialogFiller.h"
 
+#include "utils/GTUtilsDialog.h"
+
 namespace U2 {
 
 namespace GUITest_regression_scenarios {
 using namespace HI;
 
+GUI_TEST_CLASS_DEFINITION(test_7003) {
+    // 1. Ensure that 'UGENE_EXTERNAL_TOOLS_VALIDATION_BY_PATH_ONLY' is not set to "1"
+    // 2. Open "UGENE Application Settings", select "External Tools" tab
+    // 3. Add the 'dumb.sh' or 'dumb.cmd' as a Python executable
+    // 4. Check that validation fails
+
+    CHECK_SET_ERR(qgetenv("UGENE_EXTERNAL_TOOLS_VALIDATION_BY_PATH_ONLY") != "1",
```

Comment:
You need to reset the variable in the test if it was set externally:
qputenv("UGENE_EXTERNAL_TOOLS_VALIDATION_BY_PATH_ONLY", "0");